contrib/linux-kernel/zstd_compress_module.c

```diff
@@ -216,5 +222,25 @@ void zstd_register_sequence_producer(
 }
 EXPORT_SYMBOL(zstd_register_sequence_producer);
 
+size_t zstd_compress_sequences(zstd_cctx *cctx, void* dst, size_t dst_capacity,
+			       const zstd_sequence *in_seqs, size_t in_seqs_size,
+			       const void* src, size_t src_size)
+{
+	return ZSTD_compressSequences(cctx, dst, dst_capacity, in_seqs,
```

Comment:
spaces instead of tabs

Reply:
This code will be ported to the kernel automatically. The kernel coding stiles uses tabs instead of spaces:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#indentation

Reply:
oh, sorry, didn't notice it was in contrib.